server/meca/routes.js

```diff
@@ -0,0 +1,39 @@
+const logger = require('@pubsweet/logger')
+const config = require('config')
+// TODO avoid this recursive dependency
+const ManuscriptManager = require('@elifesciences/xpub-server/entities/manuscript')
+
+module.exports = app => {
+  const apiKey = config.get('meca.apiKey')
+
+  app.post('/meca-result/:id', (req, res) => {
```

Comment:
It's worth noting that this is the only REST endpoint in the project, and everything else uses the GraphQL API

Reply:
Did you have any particular way you wanted to "note" this?